### PR TITLE
HPCC-21589 include xsd files in Clienttools

### PR DESCRIPTION
**Pull request by xwang2713**
Signed-off-by: Xiaoming Wang <xiaoming.wang@lexisnexi.com> 

@kenrowland please review

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Clienttools build: 
  Linux and Mac OS: http://10.240.32.242/builds/custom/JIRA/HPCC-21589/
  The build Clienttools fails due to known error.

### CODE REVIEW

**Comment by stuartort** [comment]
@GordonSmith   @kenrowland   can you review

File: initfiles/componentfiles/CMakeLists.txt [hunk 1]
```diff
@@ -13,9 +13,10 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 ################################################################################
-
-ADD_SUBDIRECTORY(ftslave)
+if ( PLATFORM AND UNIX )
+   ADD_SUBDIRECTORY(ftslave)
+   ADD_SUBDIRECTORY(configschema)
+   ADD_SUBDIRECTORY(thor)
+   ADD_SUBDIRECTORY(launcher)
```
Comment (kenrowland) [inline initfiles/componentfiles/CMakeLists.txt]: Unless modification templates are necessary, only the configschema/xsd directory is really needed. Also, there will need to be a plugin directory added in the future for LN type builds. That work is not complete yet, however.
@xwang2713

Comment (xwang2713) [reply initfiles/componentfiles/CMakeLists.txt]: Replace "componentfiles/configxml" with "componentfiles/configschema/xsd". Builds are updated also.
@kenrowland back to you.

**Review by kenrowland — APPROVED** [review]
@xwang2713 This should be good for now.

**Comment by xwang2713** [comment]
@richardkchapman  this is ready to merge.

**Pull request merged**